corehq/motech/fhir/utils.py

```diff
@@ -68,8 +67,8 @@ def update_fhir_resource_property(case_property, fhir_resource_type, fhir_resour
 @memoized
 def load_fhir_resource_types(fhir_version=FHIR_VERSION_4_0_1, exclude_resource_types=('fhir',)):
     schemas_base_path = get_schema_dir(fhir_version)
-    all_file_names = glob.glob(os.path.join(schemas_base_path, '*.schema.json'))
-    resource_types = [os.path.basename(f).removesuffix('.schema.json') for f in all_file_names]
+    all_file_names = os.listdir(schemas_base_path)
+    resource_types = [f.removesuffix('.schema.json') for f in all_file_names if f.endswith('.schema.json')]
```

Comment:
nit: good to use `file_name` instead of `f`.

Using such variable names definitely make it hard to understand the purpose of variable later, say on sentry, in case there was an error and value of variables was sent along with stacktrace.
Also, in case you use a debugger here in this part of code, it would conflict, if "f" is used as a shortcut for some operation.

Reply:
Fixed [e7b0426](https://github.com/dimagi/commcare-hq/pull/32462/commits/e7b04266cc178c1d054358a02e9e74724d0c4cc9)